src/backend/tests/integration/components/gridgain/test_gridgain_memory_component.py

```
@@ -0,0 +1,220 @@
+import time
+
+import pygridgain
+import pytest
+from base.langflow.components.memories.gridgain import GridGainChatMemory
```

Pylint (3.3.7)

[error] No name 'gridgain' in module 'base.langflow.components.memories'

(E0611)